projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```diff
@@ -1139,6 +1139,25 @@
         bottom: 0;
     }
 
+    %igx-grid__td--bool {
+        .igx-grid__td-text {
+            height: rem(18px);
+            width: rem(18px);
+            margin: auto;
+        }
+
+        igx-icon {
+            height: rem(18px);
+            width: rem(18px);
+            font-size: rem(18px);
+            color: igx-color($palette, 'grays', 500);
+
+            &.igx-grid__td--bool-check {
```

Comment:
Same as above. Don't do this.

Reply:
This one is not created by the BEM mixins, it'a custom.

Reply:
I can tell by the name as it doesn't follow the BEM naming convention. You should use BEM to generate the class names. Doing what you did here is generally not allowed, or only allowed in some edge cases.